src/components/additem/AddProduct.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useRef } from "react";
+import styles from "./AddProduct.module.css";
+
+interface AddProductProps {
+  productImage: string;
+  productTitle: string;
+  productDescription: string;
+  productPrice: string;
+  productTags: string[];
+}
```

Comment:
P2:
AddProductProps라는 이름이 있으니 내부 속성들은 product를 빼는 것이 가독성에 더 좋을 것 같습니다.
```suggestion
interface AddProductProps {
  image: string;
  title: string;
  description: string;
  price: string;
  tags: string[];
}
```